notebooks/official/workbench/spark/spark_sample_notebook.ipynb

```diff
@@ -0,0 +1,1219 @@
+{
```

Comment:
<p>You can use preprocessed columns, <code>mono_size</code> and <code>mono_language</code> to get the average size of each language.</p><p>The <code>mono_size</code> column's bytes are kilobytes. In the following query, <code>mono_size</code> is divided by 1000 to convert it to megabyte<strong>s</strong>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/720/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>